net/netifyd/Makefile

```diff
@@ -0,0 +1,83 @@
+# Netify Agent
+# Copyright (C) 2016-2018 eGloo, Incorporated
+#
+# This is free software, licensed under the GNU General Public License v3.
```

Comment:
I would suggest licensing this Makefile under GPL-v2 like other files in this repo, otherwise you will need to also provide a GPL v3 text file along with this commit.  But I fail to see the benefit of GPL-v3 over GPL-v2 in this case.